webapp/components/tabs.tsx

```diff
@@ -63,5 +63,7 @@ type TabsProps = {
 }
 
 export const Tabs = ({ children }: TabsProps) => (
-  <ul className="flex flex-wrap items-center gap-x-2 gap-y-1">{children}</ul>
+  <ul className="flex w-full flex-wrap items-center gap-x-2 gap-y-1 [&>li>a]:w-full [&>li]:flex-1 md:[&>li]:flex-auto">
```

Comment:
I must insist: I think this can and should be moved into the tab component.

The `li` tag can add `flex-1` and `md:flex-auto`, and add `[&>a]:w-full`

```suggestion
  <ul className="flex w-full flex-wrap items-center gap-x-2 gap-y-1">
```

Reply:
Done.